### ISSUE TITLE
Beta Issue: Uncaught ReferenceError: _ is not defined

### ISSUE THREAD

**Issue by miklb** [imdone/imdone-atom#139]
Using latest Atom Beta (also with previous versions of Atom Beta & stable) 

[Enter steps to reproduce below:]

1. Open task board

**Atom Version**: 1.12.0-beta0
**Electron Version**: 1.3.6
**System**: Mac OS X 10.12.1
**Thrown From**: [imdone-atom-beta](https://github.com/innobuilt/imdone-atom-beta) package, v1.9.14

### Stack Trace

Uncaught ReferenceError: _ is not defined

```
At /Users/miklb/.atom/packages/imdone-atom-beta/lib/plugins/plugin.coffee:26

ReferenceError: _ is not defined
    at ConnectorPlugin.module.exports.ConnectorPlugin.githubProjectInfo (/Users/miklb/.atom/packages/imdone-atom-beta/lib/plugins/plugin.coffee:26:14)
    at ConnectorPlugin.module.exports.ConnectorPlugin.addMetaKeyConfig (/Users/miklb/.atom/packages/imdone-atom-beta/lib/plugins/plugin.coffee:45:20)
    at new ConnectorPlugin (/Users/miklb/.atom/packages/imdone-atom-beta/lib/plugins/plugin.coffee:19:6)
    at /Users/miklb/.atom/packages/imdone-atom-beta/lib/views/imdone-atom-view.coffee:303:22
    at /Users/miklb/.atom/packages/imdone-atom-beta/lib/services/connector-manager.coffee:45:7
    at /Users/miklb/.atom/packages/imdone-atom-beta/lib/services/connector-manager.coffee:41:7
    at /Users/miklb/.atom/packages/imdone-atom-beta/lib/services/imdoneio-client.coffee:196:7
    at Request.callback (/Users/miklb/.atom/packages/imdone-atom-beta/node_modules/superagent/lib/node/index.js:619:12)
    at /Users/miklb/.atom/packages/imdone-atom-beta/node_modules/superagent/lib/node/index.js:795:18
    at Stream.<anonymous> (/Users/miklb/.atom/packages/imdone-atom-beta/node_modules/superagent/lib/node/parsers/json.js:16:7)
    at emitNone (events.js:86:13)
    at Stream.emit (events.js:185:7)
    at Unzip.<anonymous> (/Users/miklb/.atom/packages/imdone-atom-beta/node_modules/superagent/lib/node/utils.js:112:12)
    at emitNone (events.js:91:20)
    at Unzip.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:973:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

### Commands

```
     -0:07.3.0 imdone-atom:tasks (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "markdown-format",
      "atom-prettify",
      "Zen",
      "alignment",
      "foldingtext-markdown",
      "atom-liquid-snippets",
      "nodeschool",
      "wordpress",
      "codebug",
      "marked",
      "precss",
      "travis-ci-status",
      "wordpress-api",
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-go",
      "language-java",
      "language-make",
      "language-objective-c",
      "language-toml",
      "welcome",
      "csslint",
      "linter-textlint",
      "aligner-scss",
      "linter-php",
      "linter-stylelint",
      "linter-csslint",
      "wakatime",
      "linter-scss-lint",
      "atom-phpcs",
      "linter-phpcs",
      "imdone-atom"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "themes": [
      "one-dark-ui",
      "oceanic-next"
    ]
  }
}
```

### Installed Packages

```coffee
# User
aligner, v0.22.2 (active)
aligner-php, v1.0.0 (active)
aligner-scss, v1.1.0 (inactive)
alignment, v1.1.5 (inactive)
atom-beautify, v0.29.13 (inactive)
atom-liquid-snippets, v0.3.0 (inactive)
atom-phpcs, v1.4.2 (inactive)
atom-prettify, v0.1.3 (inactive)
Atom-Syntax-highlighting-for-Sass, v0.5.0 (active)
autocomplete-wordpress-hooks, v0.2.2 (active)
autoprefixer, v3.3.0 (inactive)
codebug, v1.0.0 (inactive)
css-clean, v2.15.0 (inactive)
csscomb, v0.3.1 (inactive)
csslint, v1.1.5 (inactive)
file-icons, v1.7.20 (active)
foldingtext-markdown, v1.3.2 (inactive)
gist-it, v0.9.1 (inactive)
git-time-metric, v1.1.5 (active)
hip-light-ui-theme, v1.1.1 (inactive)
imdone-atom, v1.3.31 (inactive)
imdone-atom-beta, v1.9.14 (active)
imdone-atom-github, v0.1.11 (active)
jekyll, v1.4.2 (inactive)
json-colorer, v0.3.0 (active)
language-liquid, v0.5.1 (active)
linter, v1.11.18 (active)
linter-alex, v3.2.0 (active)
linter-csslint, v1.3.3 (inactive)
linter-eslint, v8.0.0 (active)
linter-js-yaml, v1.2.5 (active)
linter-jsonlint, v1.3.0 (active)
linter-php, v1.2.0 (inactive)
linter-phpcs, v1.5.0 (inactive)
linter-proselint, v3.2.0 (active)
linter-sass-lint, v1.7.1 (active)
linter-scss-lint, v3.0.4 (inactive)
linter-stylelint, v3.3.1 (inactive)
linter-textlint, v2.0.0 (inactive)
linter-write-good, v0.8.0 (active)
markdown-format, v2.5.1 (inactive)
markdown-writer, v2.6.1 (inactive)
marked, v0.1.10 (inactive)
merge-conflicts, v1.4.4 (active)
newbound-dark-syntax, v1.0.1 (inactive)
newbound-light-syntax, v1.2.2 (inactive)
newbound-warm-syntax, v1.0.0 (inactive)
nodeschool, v1.0.1 (inactive)
oceanic-next, v0.4.0 (active)
pigments, v0.37.0 (active)
precss, v0.3.0 (inactive)
pretty-json, v1.6.1 (active)
project-manager, v3.2.3 (active)
selection-counter, v1.0.0 (active)
selector-to-tag, v0.4.1 (active)
solarized-dark-ui, v0.3.1 (inactive)
solarized-light-ui, v0.1.0 (inactive)
sort-lines, v0.14.0 (inactive)
split-diff, v1.0.5 (inactive)
tabs-to-spaces, v1.0.2 (active)
travis-ci-status, v1.3.0 (inactive)
wordpress, v0.2.0 (inactive)
wordpress-api, v1.2.2 (inactive)
Zen, v0.16.4 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (active)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (inactive)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.13.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.4 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.202.0 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (active)
metrics, v1.0.0 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (active)
snippets, v1.0.3 (active)
spell-check, v0.68.4 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.2 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.102.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.210.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.0 (active)
language-csharp, v0.12.1 (inactive)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (inactive)
language-html, v0.46.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by piascikj** [comment]
This should be fixed in 1.9.15

**Comment by jpbourgeon** [comment]
Hi

I have a similar issue when I click on "Open files for visible tasks" 

-  Atom 1.12.2
-  Windows 10
-  imdone-atom 2.0.15

__Stack Trace__
```js
ReferenceError: _ is not defined
    at file:///C:/Users/bourgeonjp/.atom/packages/imdone-atom/lib/views/imdone-atom-view.coffee:199:18
    at Function.module.exports.Emitter.simpleDispatch (C:\Users\bourgeonjp\AppData\Local\atom\app-1.12.2\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14)
    at PluginViewInterface.module.exports.Emitter.emit (C:\Users\bourgeonjp\AppData\Local\atom\app-1.12.2\resources\app.asar\node_modules\event-kit\lib\emitter.js:129:28)
    at MenuView.module.exports.MenuView.openVisible (file:///C:/Users/bourgeonjp/.atom/packages/imdone-atom/lib/views/menu-view.coffee:126:14)
    at HTMLDivElement.<anonymous> (C:\Users\bourgeonjp\.atom\packages\imdone-atom\node_modules\space-pen\lib\space-pen.js:220:36)
    at HTMLDivElement.dispatch (C:\Users\bourgeonjp\.atom\packages\imdone-atom\node_modules\jquery\dist\jquery.js:4435:9)
    at HTMLDivElement.elemData.handle (C:\Users\bourgeonjp\.atom\packages\imdone-atom\node_modules\jquery\dist\jquery.js:4121:28)
```

**Comment by piascikj** [comment]
Thanks @jpbourgeon. It should be fixed in 2.0.16.
